src/net.c

```diff
@@ -443,6 +443,39 @@ static int hyper_set_interface_name(struct rtnl_handle *rth,
 	return 0;
 }
 
+static int hyper_set_interface_mtu(struct rtnl_handle *rth,
+				int ifindex,
+				unsigned int mtu)
+{
+	struct {
+		struct nlmsghdr n;
+		struct ifinfomsg i;
+		char buf[1024];
+	} req;
+
+	if (!rth || ifindex < 0)
+		return -1;
+
+	memset(&req, 0, sizeof(req));
+	req.n.nlmsg_len = NLMSG_LENGTH(sizeof(struct ifinfomsg));
+	req.n.nlmsg_flags = NLM_F_REQUEST;
+	req.n.nlmsg_type = RTM_SETLINK;
+	req.i.ifi_family = AF_UNSPEC;
+	req.i.ifi_index = ifindex;
```

Comment:
Need to set ifi_change to 0xFFFFFFFF as shown in man(3) rtnetlink? Looks like another reason to merge this with hyper_set_interface_name().